tests/tensorizer_loader/test_tensorizer.py

```diff
@@ -280,3 +292,196 @@ def test_vllm_tensorized_model_has_same_outputs(vllm_runner, tmp_path):
         # noqa: E501
 
         assert outputs == deserialized_outputs
+
+def test_assert_serialization_kwargs_passed_to_tensor_serializer(tmp_path):
+
+    serialization_params = {
+        "limit_cpu_concurrency": 2,
+    }
+    model_ref = "facebook/opt-125m"
+    model_path = tmp_path / (model_ref + ".tensors")
+    config = TensorizerConfig(tensorizer_uri=str(model_path),
+                              serialization_kwargs=serialization_params)
+    llm = LLM(
+        model=model_ref,
+        device="cuda",
+    )
+
+
+    def serialization_test(self, *args, **kwargs):
+        # This is performed in the ephemeral worker process, so monkey-patching
+        # will actually work, and cleanup is guaranteed so don't
+        # need to reset things
+
+        original_dict = serialization_params
+        to_compare = {}
+
+        original = tensorizer.serialization.TensorSerializer.__init__
+
+        def tensorizer_serializer_wrapper(self, *args, **kwargs):
+            nonlocal to_compare
+            to_compare = kwargs.copy()
+            return original(self, *args, **kwargs)
+
+        tensorizer.serialization.TensorSerializer.__init__ = tensorizer_serializer_wrapper
+
+        tensorizer_config = TensorizerConfig(**kwargs.get("tensorizer_config"))
+        assert tensorizer_config is not None
+        self.save_tensorized_model(
+            tensorizer_config=tensorizer_config, )
```

Comment:
Silly spacing. You can ignore this if this will be resolved by an automatic formatter pass.
```suggestion
        self.save_tensorized_model(tensorizer_config=tensorizer_config)
```